### PR TITLE
Add wsgi handler for status report

### PR DESCRIPTION
**Pull request by jichenjc**
Add status healthy wsgi 

GET /healthy 
is supported so that we can report such
```
# curl localhost:8080/healthy
{"totalSuccess": 0, "totalFail": 0, "lastSuccess": "", "lastFail": "", "continueousFail": 0, "healthy": true}
```

Signed-off-by: jichenjc <jichenjc@cn.ibm.com>

### CODE REVIEW

File: doc/source/parameters.yaml [hunk 1]
```diff
@@ -1303,3 +1303,39 @@ fcp_connections:
   in: body
   required: true
   type: integer
+totalSuccess:
```
Comment (bjhuangr) [inline doc/source/parameters.yaml:1306]: smapi_total_success ?

File: doc/source/parameters.yaml [hunk 2]
```diff
@@ -1303,3 +1303,39 @@ fcp_connections:
...
+  description: |
+    The total success calls for SMAPI.
+  in: body
+  required: true
+  type: integer
+totalFail:
```
Comment (bjhuangr) [inline doc/source/parameters.yaml:1312]: smapi_total_fail ?

File: doc/source/parameters.yaml [hunk 3]
```diff
@@ -1303,3 +1303,39 @@ fcp_connections:
...
+  in: body
+  required: true
+  type: string
+continueousFail:
+  description: |
+    The count of continueous fail.
+  in: body
+  required: true
+  type: integer
+healthy:
```
Comment (bjhuangr) [inline doc/source/parameters.yaml:1336]: smapi_healthy

File: doc/source/restapi.rst [hunk 4]
```diff
@@ -88,6 +88,37 @@ you can think as a combination of username and password.
 
   Please refer to :ref:`TokenUsage` to get more details.
 
+Healthy
```
Comment (bjhuangr) [inline doc/source/restapi.rst:91]: SMAPI Healthy

File: doc/source/restapi.rst [hunk 5]
```diff
@@ -88,6 +88,37 @@ you can think as a combination of username and password.
...
+=======
+
+Report healthy of SMAPI
+-----------------------
+
+**GET /healthy**
```
Comment (bjhuangr) [inline doc/source/restapi.rst:97]: `/smapi-healthy`

File: zvmsdk/sdkwsgi/handlers/healthy.py [hunk 6]
```diff
@@ -0,0 +1,36 @@
+# Copyright 2017,2018 IBM Corp.
```
Comment (bjhuangr) [inline zvmsdk/sdkwsgi/handlers/healthy.py:1]: Copyright 2022IBM Corp.

**Comment by jichenjc** [comment]
@bjhuangr I changed the healthy to smapi-healthy
but the others kept (e.g lastSuccess as it anyway return the smapi healthy already)

**Review by bjhuangr — APPROVED** [review]

**Pull request merged**